### DISCUSSION TITLE
Como ter um mapa com a mesma qualidade da Ficha de Nimb?

### DISCUSSION THREAD

**Discussion by ZillinNava** [YuriAlessandro/gerador-ficha-tormenta20#374] [Perguntas e Respostas]
Eu gostaria de saber como vocês conseguiram colocar o mapa de Arton com tanta qualidade em um site, Como eu acabei dizendo na ´´postagem´´ anterior, eu gostaria de poder adicionar com algumas coisas ao mapa

**Comment by YuriAlessandro** [comment]
Para gerar esse mapa foi necessário alguma programação. Todo o código está disponível aqui: https://github.com/YuriAlessandro/artonMap

Se você manja de programação em Python e Javascript, deve ser bem tranquilo entender como funciona. Se não, é bem mais complicado.

Que tipo de coisa você quer adicionar ao mapa?

**Reply by ZillinNava** [reply]
Não saberia dizer especificamente, mas coisas para representar novas localizações , cidades, rios e etc...

Só mais uma coisa, você pretende expandir o mapa? eu tenho o livro do Atlas de Arton e acabei notando que o deserto da perdição tem algumas partes faltando.

**Reply by YuriAlessandro** [reply]
Não tenho planos para isso atualmente. Meu foco atual vai ser em melhorar o gerenciamento de pontos próprios, com uma lista e permitir edição e tal.

É um bocado dificil colocar descrição de absolutamente tudo que existe sozinho. Quando eu to lendo material e percebo que ta faltando, eu coloco. Você pode usar esse tópico para sugerir novos locais: https://github.com/YuriAlessandro/gerador-ficha-tormenta20/discussions/367

Sempre que possível eu vou atualizando.

**Reply by ZillinNava** [reply]
Claro! valeu pela atenção amigo, um bom dia para você
